build.gradle

```diff
@@ -378,6 +379,13 @@ tasks.withType(Checkstyle) {
     }
 }
 
+spotless {
+  java {
+    // note: you can use an empty string for all the imports you didn't specify explicitly, and '\\#` prefix for static imports
+    importOrder('java', 'javax', '', 'com.amazon', 'org.opensearch', '\\#')
```

Comment:
Open to suggestions if we'd like to change this ordering around.

Reply:
you can use an empty string for all the imports you didn't specify explicitly, this way we are putting all `com.amazon` and `org.opensearch` imports below any other libraries